docs/user-guide/admin-client/consumer.rst

```diff
@@ -76,7 +76,7 @@ bindings are waiting for that remote action to take place.
     Available Arguments:
 
       --fields   - comma-separated list of consumer fields; if specified only the
-                   given fields will be displayed. Example: "id,display_name"
+                   given fields will be displayed. Example: "id,display_name".
```

Comment:
I updated the docs the reflect the exact behavior/output of CLI
```
pulp-admin consumer list --help


Command: list
Description: lists a summary of consumers registered to the Pulp server

Available Arguments:

  --fields   - comma-separated list of consumer fields; if specified only the
               given fields will be displayed. Example: "id,display_name".
  --bindings - if specified, the bindings information is displayed
  --details  - if specified, all of the consumer information is displayed
```
So I will just leave it as it is to make docs info  100% same like in CLI